hr_employee_firstname/views/hr_view.xml

```diff
@@ -6,9 +6,9 @@
         <field name="model">hr.employee</field>
         <field name="inherit_id" ref="hr.view_employee_form"/>
         <field name="arch" type="xml">
-            <label for="name" position="replace">
+            <xpath expr="//label[@for='name']" position="replace">
```

Comment:
if you use 'replace' you need to change the priority of the view definition to 100 or more to avoid breaking other views depending on the label for="name" anchor.

Reply:
@gurneyalex you're right. The 'replace' position was there before my **PR** actually. What do you suggest to correct this?

Reply:
change the priority of the view (this field: https://github.com/odoo/odoo/blob/8.0/openerp/addons/base/ir/ir_ui_view.py#L126)

Reply:
see suggestion below